lib/rester/service/resource/params.rb

```diff
@@ -21,7 +21,7 @@ def initialize(opts={}, &block)
       # Whether or not validation will be done strictly (i.e., only specified
       # params will be allowed).
       def strict?
-        !!options[:strict]
+        !!options.fetch(:strict, true)
```

Comment:
Hmm... I think I like that better because then it'll show up if you call `.options`.

But how about:
```ruby
DEFAULT_OPTS = { strict: true }.freeze
@options = DEFAULT_OPTS.merge(opts).freeze
```